kratos/solving_strategies/schemes/residual_based_bdf_custom_scheme.h

```diff
@@ -542,6 +544,7 @@ class ResidualBasedBDFCustomScheme
         Parameters default_parameters = Parameters(R"(
         {
             "name"                  : "ResidualBasedBDFCustomScheme",
+            "domain_size"           : 3,
```

Comment:
can u detect this automatically? 

all the nodes must have the same dofs, so if the first dof has only DISPLACEMENT_X but not Y than dim=1,
if it has DISPLACEMENT_Z -> 3...

Reply:
this should go out. You are detecting it automatically

Reply:
Done, but must be done in the constructor, so it is corrected later

Reply:
ok i'll pass on this. I mark the issue as API Breaker to take this into account